css/registrationFrom.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="bg">
+<head>
+    <title>Регистрация</title>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <link rel="stylesheet" type="text/css" href="registrationForm.css">
+</head>
+</head>
+<body>
+    <form method="POST">
+        <h4>РЕГИСТРАЦИЯ</h4>
+        <input type="text" class="fieldsStyle" placeholder="E-mail">
+        <br>
+        <input type="text" class="fieldsStyle" placeholder="Име">
+        <br>
+        <input type="text" class="fieldsStyle" placeholder="Фамилия">
+        <br>
+        <input type="password" class="fieldsStyle" placeholder="Парола">
+        <br>
+        <input type="password" class="fieldsStyle" placeholder="Повторна парола">
+        <br>
+        <input type="submit" class="fieldsStyle" id="registrationButon" value="Регистрация">
+    </form>
+</body>
+</html>
```

Comment:
I meant adding the snapshot as a comment in the PR, please remove it from commit and add as comment. I will also fix this on the slides. Thanks!